modules/android/pages/gs-install.adoc

```diff
@@ -271,6 +272,13 @@ dependencies {
 }
 ----
 
```

Comment:
There's a "+" character missing in the existing text, between lines 262 and 263,
which (in GH preview) is affecting the treatment of the ordered list, and causing your new content below to start the count again.

Reply:
![Uploading Screenshot 2024-09-04 at 16.58.04.png…]()

Reply:
Fixed so ordering is displayed correctly.